ui_framework/components/info_button/_info_button.scss

```diff
@@ -0,0 +1,13 @@
+.kuiInfoButton {
+  font-size: 16px;
+  background-color: transparent;
+  color: $infoButtonColor;
+  cursor: pointer;
+  border: none;
+  padding: 0;
+
+  &:hover,
+  &:active {
+    color: $infoButtonColor;
```

Comment:
The suggestion seems fine. I did a search to find an existing variable but I guess I didn't uncover it. Thanks!